README.md

```diff
@@ -24,7 +24,7 @@
 See [DEVELOPMENT](.github/DEVELOPMENT.md) for information about code style,
 development process, and guidelines.
 
-See [CONTRIBUTING](.github/CONTRIBUTING.md) for contribution requirements.
```

Comment:
Do we need a link to CONTRIBUTING.md somewhere?

let's keep the link here, and let the CONTRIBUTING.md delegate to the website

Reply:
> Then you have two steps to follow, instead of one

i am aware, this is a drawback

> since there's almost no content in the current CONTRIBUTING.md file.

today

i am concerned someone may add some content in the future, and it will be invisible

("single source of truth")

Reply:
> someone may add some content in the future, and it will be invisible

Good point, I agree. Should we consider removing the file then?

Reply:
my preference is to keep it, with a link to the website.

Reply:
Two steps are better than none, so to get this merged sooner I removed changes in `README.md`.